packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -34,7 +36,7 @@ export interface AtomicSearchBoxOptions {
 @Component({
   tag: 'atomic-search-box',
   styleUrl: 'atomic-search-box.pcss',
-  shadow: true,
+  shadow: false,
```

Comment:
Unfortunately, it doesn't look Android Talkback properly supports the shadow DOM.

Reply:
Is there a way the suggestions could be a separate component encapsulated in a ShadowDOM? This could also make them re-usable.